src/libraries/System.Security.Cryptography.Xml/src/System/Security/Cryptography/Xml/XmlDsigXPathTransform.cs

```diff
@@ -48,7 +48,7 @@ public override void LoadInnerXml(XmlNodeList nodeList)
                     if (elem.LocalName == "XPath")
                     {
                         _xpathexpr = elem.InnerXml.Trim(null);
-                        XmlNodeReader nr = new XmlNodeReader(elem);
+                        using XmlNodeReader nr = new XmlNodeReader(elem);
```

Comment:
Does this actually help anything? What resource is being leaked without it? I think the answer is that Dispose here is effectively a nop, in which case we don't want to churn the code. We only take critical fixes into this library:
https://github.com/dotnet/runtime/blob/main/src/libraries/System.Security.Cryptography.Xml/README.md#contribution-bar